kernel/kernel-impl/src/main/java/org/sakaiproject/email/impl/BasicEmailService.java

```diff
@@ -135,6 +135,9 @@
 	/** Whether to turn on mail debugging */
 	public static final String MAIL_DEBUG = "mail.debug";
 
+	public static final String MAIL_OURDOMAIN = "mail.ourdomain";
```

Comment:
Okay, so I would recommend something like "Rutgers Sakai <postmaster@sakai.rutgers.edu>"

This would be ui.service + "<postmaster@" + ServerConfigurationService.getServerName()